dev/build/makepack-howto.md

```diff
@@ -0,0 +1,142 @@
+# Dolibarr Makepack How To
+
+This documentation describe steps to build a BETA or RELEASE versions of Dolibarr.
+There is a chapter for BETA version and a chapter for RELEASE version.
+
+
+## Prerequisites on Linux
+
+Prerequisites to build the tgz, debian and rpm packages:
+
+`apt-get install perl tar dpkg dpatch p7zip-full rpm zip php-cli`
+
+Prerequisites to build autoexe DoliWamp package from Linux (solution seems broken since Ubuntu 20.04+):
+
+`apt-get install wine q4wine`
+
+* Launch "wine cmd" to check a drive Z: pointing to / exists.
+* Install InnoSetup
+
+  For example by running isetup-5.5.8.exe (https://www.jrsoftware.org)  https://files.jrsoftware.org/is/5/
+
+* Install WampServer into "C:\wamp64" to have Apache, PHP and MariaDB
+
+  For example by running wampserver3.2.6_x64.exe (https://www.wampserver.com).
+See file dev/build/exe/doliwamp.iss to know the doliwamp version currently setup.
+
+* Add path to ISCC into PATH windows var:
```

Comment:
.. into the windows PATH environment variable (look for "Modify environment variable" from the run menu in windows (in your language):